media-autobuild_suite.bat

```diff
@@ -1373,6 +1373,40 @@ if not exist %instdir%\mintty.lnk (
     del %build%\setlink.vbs
 )
 
+rem createFolders
```

Comment:
This commit looks like new code is being added instead of being moved/refactored somewhere else. You can join the commits removing the old code and adding the new one to make it more atomic.

Reply:
I honestly did have it like that, but, I had too many changes, that I reset to upstream and did commit -p

Reply:
I'll try again to see if I can combine changes into logical commits

Reply:
Although I squashed the commit that actually remove the code from below, I can't seems to figure out a good way to separate out the update.log part and pacman. Need to learn more about git

Reply:
I realized I just needed to rebase and edit and reset and recommit